src/cpp/cpp_typecheck_constructor.cpp

```diff
@@ -295,7 +294,7 @@ void cpp_typecheckt::default_assignop(
   cpctor.operands().push_back(exprt(ID_cpp_declarator));
   cpctor.add_source_location()=source_location;
 
-  cpp_declaratort &declarator=(cpp_declaratort&) cpctor.op0();
+  cpp_declaratort &declarator = (cpp_declaratort &)cpctor.op0();
```

Comment:
Looks like a whitespace-only change, but maybe we can actually get rid of the C-style cast?

Reply:
done